### DISCUSSION TITLE
Bot got kicked, can't invite anew

### DISCUSSION THREAD

**Discussion by sverit** [Cog-Creators/Red-DiscordBot#5123] [Q&A]
EDIT: Solved, it was a problem with the rights, nevermind :)

Hi,
#
my friend kicked the RedDiscordbot User from our server, If I try to add the bot again via its invite URL nothing happens. Any ideas?

Thanks :)

**Comment by zarigata** [comment]
me as well.... i was abbout to ask a question and i was banned
